docs/quick_start.rst

```diff
@@ -21,3 +21,22 @@ Output:
  * output.nwb: NWB file including spike times
  * log.txt: run log
  * qc_figs: index.html includes cell figures and feature table and sweep.html includes sweep figures
+
+
```

Comment:
We already have a page describing how to download data on the Data Access page https://github.com/AllenInstitute/ipfx/pull/450. We should not duplicate the content to avoid needing to update two pages in the future. I believe on the quick_start.rst we should outline the steps to get started, but refer the user to the data access page for specific  download instructions. Same with the installation of ipfx. 
Also I believe, the comment in the issue #309 should be fully answered on the quick_start.rst and we would just refer the user to the updated quick_start page rather than providing quick start instructions for a particular user.

Reply:
remove the duplicated downloaded info and added guideline with an example as suggested.